controllers/storagecluster/cephblockpools.go

```diff
@@ -18,11 +18,22 @@ type ocsCephBlockPools struct{}
 // newCephBlockPoolInstances returns the cephBlockPool instances that should be created
 // on first run.
 func (r *StorageClusterReconciler) newCephBlockPoolInstances(initData *ocsv1.StorageCluster) ([]*cephv1.CephBlockPool, error) {
+	mirroringPeerSpec := &cephv1.MirroringPeerSpec{}
 	var mirroringSpec cephv1.MirroringSpec
+
 	if initData.Spec.Mirroring.Enabled {
 		mirroringSpec.Enabled = true
 		mirroringSpec.Mode = "image"
+
+		// ensures that peer cluster secret is exist and appending into cephBlockPool
+		secretName := initData.Name + "-" + "token"
```

Comment:
```suggestion
		secretName := "cluster-peer-token" + initData.Name
```
to keep it consistent with CephCluster token name.

Reply:
"cluster-peer-token-" + initData.Name?

Reply:
Yes.